### PR TITLE
Correction to markdown syntax for list format

### PR DESCRIPTION
**Pull request by chris18890**
I noticed that a few list items on https://docs.microsoft.com/en-us/powershell/module/remoteaccess/install-remoteaccess?view=win10-ps weren't rendering correctly, this should make them consistent with the other items in the respective lists.

### CODE REVIEW

File: docset/windows/remoteaccess/Install-RemoteAccess.md [hunk 1]
```diff
@@ -769,7 +769,7 @@ Accept wildcard characters: False
 Specifies the type of VPN installation.
 The acceptable values for this parameter are:
 - Vpn.
```
Comment (JohanFreelancer9) [inline docset/windows/remoteaccess/Install-RemoteAccess.md:771]:
```suggestion
- Vpn
```

File: docset/windows/remoteaccess/Install-RemoteAccess.md [hunk 2]
```diff
@@ -769,7 +769,7 @@ Accept wildcard characters: False
 Specifies the type of VPN installation.
 The acceptable values for this parameter are:
 - Vpn. 
- -- VpnS2S. 
+- VpnS2S.
```
Comment (JohanFreelancer9) [inline docset/windows/remoteaccess/Install-RemoteAccess.md:772]:
```suggestion
- VpnS2S
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please also remove the two periods as indicated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
Thanks for your contribution!

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**